src/qgis_stac/ui/qgis_stac_widget.ui

```diff
@@ -20,8 +20,14 @@
    <string>STAC Search</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
-   <item row="0" column="1">
+   <item row="0" column="0">
```

Comment:
There are some changes in this file that has cause the UI layout to be misaligned, see the below screenshot.
![ui_issue](https://user-images.githubusercontent.com/2663775/142835947-8f003723-ad3c-4eee-815b-09fb3565d1be.png)

Reply:
@Fanevanjanahary this issue still present from the latest changes, can you double check it and fix it, thanks?

Reply:
Also we need to make sure the UI is responsive